### PR TITLE
[fix] Retention line graph modal

### LINKED ISSUE #7619: Retention line graph modal white screen
## Bug description

*Please describe.*  
- trying to click on a datapoint for retention line graph doesn't work. It whitescreens
*If this affects the front-end, screenshots would be of great help.*  

## Expected behavior
The modal should appear

## How to reproduce

1. Go to retention
2. Make the chart type line graph
3. click on a datapoint

## Environment

- [ ] PostHog Cloud
- [ ] self-hosted PostHog (ClickHouse-based), version/commit: _please provide_
- [ ] self-hosted PostHog (Postgres-based, legacy), version/commit: _please provide_

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes

*Please describe.*  
- fixes #7619 
- line graph will now return the same modal as table
*If this affects the frontend, include screenshots.*  

## How did you test this code?

*Please describe.*
- clicked both line graph points and table points to make sure they return the same thing

### CODE REVIEW

File: frontend/src/scenes/retention/RetentionLineGraph.tsx [hunk 1]
```diff
@@ -55,43 +58,31 @@ export function RetentionLineGraph({
                     dashboardItemId
                         ? undefined
                         : (point) => {
-                              const { index } = point
```
Comment (rcmarron) [inline frontend/src/scenes/retention/RetentionLineGraph.tsx:58]: Super nit: feels a bit weird to have `points` inside of `point`. I didn't look much past the surface level here, but I wonder if `clickedCoordinate` or something else may be a better name. Ignore me if this is following patterns elsewhere/is a bigger refactor.

**Review by rcmarron — APPROVED** [review]
LGTM - definitely fixes the crash.

I think there is still some work to do to get the modal to work for group support. Probably a different PR, but a couple of things that I noticed:
* We use the word "persons"/"people" for "Load more people" and "No persons during this period." even when it's showing groups
* Looks like the code flips between `actor` and `person` terminology. Makes it a bit tricky to follow.
* Seems like there's a bug where we append an extra row to the table when showing groups
image

File: frontend/src/scenes/retention/RetentionLineGraph.tsx (hunk 1 above)
Comment (EDsCODE) [reply frontend/src/scenes/retention/RetentionLineGraph.tsx:58]: Yeah, the payload types are a bit weird. I change dit slightly to match the other usages but could use an overall cleanup

**Comment by EDsCODE** [comment]
Changed the labels. Couldn't find the extra row problem?

**Comment by rcmarron** [comment]
> Changed the labels. Couldn't find the extra row problem?

Oooo just looked into the extra row issue. Looks like I had one rogue `pageview` event in my local clickhouse that didn't have any group properties. I had been running an old `posthog-js` yesterday, so probably a result of that. Sorry about that.

**Pull request merged**